cfg.lex

```diff
@@ -267,6 +267,7 @@ MCAST_LOOPBACK		"mcast_loopback"
 MCAST_TTL			"mcast_ttl"
 TOS					"tos"
 DISABLE_DNS_FAILOVER  "disable_dns_failover"
+REDACTING_PII "redacting_pii"
```

Comment:
This is picking nits for sure, but I don't like the continuous tense here. It's not consistent with other settings. `REDACT_PII` would be better. Not required to change, just my opinion.

Reply:
Updating this now thanks Ben!